spec/features/team/reset_2fa_spec.rb

```diff
@@ -0,0 +1,133 @@
+describe "Reset two factor authentication", type: :feature do
+  let(:organisation) { create(:organisation) }
+  let(:org_admin_user_1) { create(:user, organisations: [organisation]) }
+  let(:org_admin_user_2) { create(:user, organisations: [organisation]) }
+  let(:org_admin_user_3) { create(:user, organisations: [organisation]) }
+
+  let(:super_admin_organisation) { create(:organisation, super_admin: true) }
+  let(:super_admin_user) { create(:user, organisations: [super_admin_organisation]) }
```

Comment:
you can use `create(:user, :new_admin)` and you're done. Or `create(:user, :superadmin)` to have the org created but in both cases the 2FA is setup straight away I believe (see the user factory).